resources/tentative/interactive-catalog-navigation/scripts/test.js

```diff
@@ -0,0 +1,148 @@
+const columNum = 6;
+// you can customize table rows by changing 'rowNum' value
+const rowNum = 100;
+const dataMap = new Map([
+    [
+        "Pizza",
+        {
+            category: "Food",
+            price: "$5",
+            popularity: "80%",
+            rating: 4,
+        },
+    ],
+    [
+        "Wine",
+        {
+            category: "Drink",
+            price: "$25",
+            popularity: "85%",
+            rating: 5,
+        },
+    ],
+    [
+        "Ice cream",
+        {
+            category: "Dessert",
+            price: "$3",
+            popularity: "90%",
+            rating: 1,
+        },
+    ],
+    [
+        "Barbecue",
+        {
+            category: "Food",
+            price: "$5",
+            popularity: "95%",
+            rating: 2,
+        },
+    ],
+    [
+        "Donut",
+        {
+            category: "Dessert",
+            price: "$2",
+            popularity: "98%",
+            rating: 3,
+        },
+    ],
+]);
+
+function generateTableBody(name) {
```

Comment:
Firefox and Chrome spends roughly 1/4 of the time in this method, not sure that's the intended purpose.

Reply:
The generated table contains some heavy CSS effects and animations, which reflects the purpose of the workload. However generateTableBody function sets many innerHTML which may consume much time in HTML parsing. We can try to use a big innerHTML instead of a lot of small ones